packages/launchpad/src/global/GlobalPageHeader.vue

```diff
@@ -5,30 +5,59 @@
     class="min-w-200px w-80% flex-grow"
     :placeholder="t('globalPage.searchPlaceholder')"
   />
+  <input
+    ref="fileInputRef"
+    type="file"
+    class="hidden"
+    webkitdirectory
+    webkit-relative-path
```

Comment:
I assume these are here for cross browser compat. Let's make sure they aren't accidentally deleted.

```javascript
cy.get('input[file]')
  .should.have.attr('webkitdirectory')
  .and.have.attr('webkit-relative-path')
```